### DISCUSSION TITLE
Merging 2 NFS mounts each of 100TB capacity

### DISCUSSION THREAD

**Discussion by gpradeepjain** [trapexit/mergerfs#912] [Q&A] [answered]
Hi,

I am having 2 NFS mounts each of 100TB capacity i.e 100+100 = 200 TB. I am planning to mount these 2 NFS on Kubernetes POD and use mergerfs to show them as a single filesystem.  Then put Nginx in front of the new union folder as a file explorer. Just want to understand do you see any problems or pitfalls in this design, as the size of the mount is large i.e. 100TB each. Will mergerfs have any problem if the capacity of mount is large as in my case?

**Comment by trapexit** [comment, marked as answer]
No. Why would mergerfs have issues with the capacity? 200TB is not that much storage and well within the 64bit limits of modern POSIX filesystem size and offset values. The biggest possible issue is using NFS as it's not fully POSIX compliant and has some known complications when interacting with FUSE but that depends entirely on your usage patterns.

**Reply by gpradeepjain** [reply]
Can you help me with known complications  you are talking about? I just use it to see 2 NFS mounts as 1 and nothing more than it. No writes on the merged folder.

**Reply by trapexit** [reply]
There are details in the documentation. If you're putting mergerfs in front of rather than behind NFS and only reading... you should be fine.

**Reply by gpradeepjain** [reply]
Thanks. So my design/settings are fine and should be good for production? So no complexity involved.

**Reply by trapexit** [reply]
Outside normal concerns for NFS, yes, it's fine.

**Reply by gpradeepjain** [reply]
I am using via this command config `mergerfs -o allow_other,use_ino` any other config to add for NFS?

**Reply by trapexit** [reply]
There really isn't anything not in the docs regarding NFS. Your not exporting mergerfs through NFS so all your options are preference.

**Reply by gpradeepjain** [reply]
Do i need to add any kind of monitoring on the fuse itself? Do you see any scenario where it may hang or break.

**Reply by trapexit** [reply]
I don't know what you mean. FUSE is a kernel driver. What exactly is there to monitor?
